README.md

```diff
@@ -84,112 +149,114 @@ Value DESCRIPTION (.*)
-
- - OS in alphabetical order
- - Command in length order
- - When length is the same, use alphabetical order of command name
- - Keep space between OS's
-
-Example:
-
-```
-Template, Hostname, Platform, Command
-
-**same os, same length, used alphabetical order of command name**
-arista_eos_show_mlag.template, .*, arista_eos, sh[[ow]] ml[[ag]]
-arista_eos_show_vlan.template, .*, arista_eos, sh[[ow]] vl[[an]]
-
-**os in alphabetical order and space between cisco_asa and arista_eos**
-cisco_asa_dir.template,  .*, cisco_asa, dir
-
-**same os, command length different and space between cisco_asa and cisco_ios**
-cisco_ios_show_capability_feature_routing.template,  .*, cisco_ios, sh[[ow]] cap[[ability]] f[[eature]] r[[outing]]
-cisco_ios_show_interface_transceiver.template, .*, cisco_ios, sh[[ow]] int[[erface]] trans[[ceiver]]
-cisco_ios_show_cdp_neighbors_detail.template, .*, cisco_ios, sh[[ow]] c[[dp]] neig[[hbors]] det[[ail]]
-```
-## Updating/Fixing Existing Templates
-When either fixing a bug within a template or adding additional **Values** to be captured additional tests should be added for the template on top of the already existing templates to ensure backwards compatibility and that the new data is being parsed correctly.
+### Updating/Fixing Existing Templates
+When either fixing a bug within a template or adding additional **Values** to be captured, additional test files should be added to ensure backwards compatibility and that the new data is being parsed correctly.
 
 To add additional raw/parsed tests for a command:
-- Navigate to `./tests/{{ vendor_name }}/{{ command_name }}/`
-- Create new `.raw` and `.parsed` files within the directory, but append the next number up
-        - Existing raw: `./tests/cisco_ios/show_clock/cisco_ios_show_clock1.raw`
-        - New raw: `./tests/cisco_ios/show_clock/cisco_ios_show_clock2.raw`
-        - Existing parsed: `./tests/cisco_ios/show_clock/cisco_ios_show_clock1.parsed`
-        - New parsed: `./tests/cisco_ios/show_clock/cisco_ios_show_clock2.parsed`
-
-### Testing
-You can test your changes locally within your Git branch before submitting a PR. This can be done by installing **tox** within your environment using pip and then running **tox** within the **ntc-templates** root directory:
-```
-(main) ~/ntc-templates[update-readme !]:~$ tox
-GLOB sdist-make: /home/myohman/ntc-templates/setup.py
-py36 inst-nodeps: /home/myohman/ntc-templates/.tox/dist/ntc_templates-1.1.0.zip
+- Navigate to `./tests/{{ vendor_os }}/{{ command_name }}/`
+- Create new `.raw` and `.parsed` files within the directory, preferrably with a name identifying why the data is unique:
+  * Existing raw: `./tests/cisco_ios/show_version/cisco_ios_show_version.raw`
+  * New raw: `./tests/cisco_ios/show_version/cisco_ios_show_version_stack_platforms.raw`
+  * Existing parsed: `./tests/cisco_ios/show_version`/cisco_ios_show_version`.parsed`
```

Comment:
Remove the **`** so it properly formats the string since it currently has **/cisco_ios_show_version** unformatted.